Testnet/MintableToken/MintableToken/MintableToken.cs

```diff
@@ -14,17 +14,20 @@ public class MintableToken : SmartContract, IStandardToken256, IMintable, IBurna
     /// <param name="totalSupply">The total token supply.</param>
     /// <param name="name">The name of the token.</param>
     /// <param name="symbol">The symbol used to identify the token.</param>
-    /// <param name="decimals">The amount of decimals for display and calculation purposes.</param>
-   public MintableToken(ISmartContractState smartContractState, UInt256 totalSupply, string name, string symbol,
-        byte decimals) : base(smartContractState)
+    /// <param name="nativeChain">The blockchain name of the token's native chain.</param>
+    /// <param name="nativeAddress">The contract address of the token's native blockchain if available.</param>    
+    public MintableToken(ISmartContractState smartContractState, UInt256 totalSupply, string name, string symbol,
+         string nativeChain, string nativeAddress) : base(smartContractState)
     {
         this.TotalSupply = totalSupply;
         this.Name = name;
         this.Symbol = symbol;
-        this.Decimals = decimals;
         this.Owner = Message.Sender;
```

Comment:
When you set a owner even initialy, it is right practice to log it 
So you can log it at the end of constructor. 
`Log(new OwnershipTransferedLog { PreviousOwner = Address.Zero, NewOwner = Message.Sender });`

Reply:
Done.